### PR TITLE
[TG-79][feat] Implement adding new sensor

### PR DESCRIPTION
**Pull request by artem713**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/13" target="_blank">ReviewNinja

File: src/app/components/device-list/device-list.html [hunk 1]
```diff
@@ -22,5 +22,6 @@
     </tr>
     </tbody>
 </table>
+<button (click)="add()">Add sensor</button>
```
Comment (yurii-sorokin) [inline src/app/components/device-list/device-list.html]: Why you do not use router link?

File: src/app/sensor-detail/sensor-detail.service.js [hunk 2]
```diff
@@ -3,14 +3,17 @@ import ShHttpService from '../sh-http/sh-http.service.js';
...
         this.http = http;
     }
+
     get(id) {
         return this.http.get(`/sensors/${id}`);
     }
 
-    save(sensor) {
-        return this.http.put(`/sensors/${sensor._id}`, sensor);
+    save(sensor, update) {
```
Comment (yurii-sorokin) [inline src/app/sensor-detail/sensor-detail.service.js]: !fix Why? Create update method instead.

File: src/app/sensor-detail/sensor.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ export default class Sensor {
             mqttId: '',
             type: '',
             description: '',
-            status: false
+            status: true
```
Comment (yurii-sorokin) [inline src/app/sensor-detail/sensor.js:7]: I know it's not your fail, but status that contains true/false...

File: src/app/sensor-detail/sensor-detail.service.js (hunk 2 above)
Comment (artem713) [inline src/app/sensor-detail/sensor-detail.service.js]: !fixed

**Pull request merged**